tools/utility/dockerise/platform.py

```diff
@@ -38,7 +38,7 @@ def selected(repository):
         else:
             print("WARNING There are multiple platforms with the same image tag.")
             print("        The possible tags are [{}]".format(", ".join(names)))
-            platform = list(sorted(names))[0]
```

Comment:
Very curious what prompted this change

Reply:
Likely an indexed access, what is returned by sorted is an iterator, not a list.
So previously `platform[0]` would have been invalid

Reply:
[the python docs](https://docs.python.org/3/library/functions.html#sorted) say that it returns a new list though?
Based on that, it seems to me that this cast to a list is simply just redundant.

Reply:
I was getting an error when I wrote it, now I cannot reproduce the error :slightly_frowning_face:. It is redundant code though, so I'm going to leave it in the pr.